CHANGES.rst

```diff
@@ -4,7 +4,8 @@ ChangeLog
 +------------+---------------------------------------------------------------------+------------+
 | Version    | Description                                                         | Date       |
 +============+=====================================================================+============+
-| *TBC*      | * Allow a wider range of SPI bus speeds                             |            |
+| **2.0.0**  | * Improved framebuffer performance (breaking change)                | TBC        |
```

Comment:
do we have an upgrade document where breaking changes can be described in more detail?

Reply:
Not really ... this is really an internal change to the implementation of luma.core and its dependent projects, rather than a material change to the public interface.

I have added things like the following to the "user-facing" API docs: https://github.com/rm-hull/luma.oled/pull/310/files#diff-b235326d22f35fd32f51120f8cccb66f7440e2f26997be86709b3afb8e29ce8aR343 and if there are specific instances where someone explicitly passing a string of "full_frame" or "diff_to_previous" it will show a runtime warning, e.g.: https://github.com/rm-hull/luma.oled/pull/310/files#diff-b235326d22f35fd32f51120f8cccb66f7440e2f26997be86709b3afb8e29ce8aR841-R849

Reply:
> this is really an internal change to the implementation of luma.core 

I would stop calling it a breaking change in that case. maybe bckwards incompatible or just omit it?